stubs/app/Actions/Jetstream/DeleteUserWithTeams.php

```diff
@@ -36,7 +36,7 @@ public function delete($user)
     {
         DB::transaction(function () use ($user) {
             $this->deleteTeams($user);
-
+            $user->deleteProfilePhoto();
```

Comment:
Could check before the deletion

```suggestion
            if (\Laravel\Jetstream\Features::managesProfilePhotos()) {
                $user->deleteProfilePhoto();
            }
```